packages/mds-types/tests/transformers/device_transformers_spec.ts

```diff
@@ -0,0 +1,68 @@
+import assert from 'assert'
+import { uuid, now } from '@mds-core/mds-utils'
+import { Device_v0_4_1 } from '../../transformers/@types'
+import { Device_v1_0_0 } from '../../index'
```

Comment:
I think we should try to keep these in a similar level, what if we have a section of `mds-types` which defines the new types for each version, and then the main `index.ts` for `mds-types` could just alias to the latest? So we could have `mds-types/versions/0.4.1` and `mds-types/versions/1.0.0`, for example

Reply:
agree

Reply:
I don't think we should, because the latest versions will have dependencies on other types in the main `index.ts`, and that would create a circular dependency if the alias is in the main `index.ts`.